src/PureCloud.Client/Apis/TelephonyProvidersEdgeApi.cs

```diff
@@ -269,25 +281,1394 @@ public async Task<ExtensionPool> CreateExtensionPoolAsync(ExtensionPool body, Ca
 
     public async Task<ExtensionPool> UpdateExtensionPoolAsync(string extensionPoolId, ExtensionPool body, CancellationToken cancellationToken = default)
     {
-        ArgumentException.ThrowIfNullOrEmpty(nameof(extensionPoolId), extensionPoolId);
+        ArgumentException.ThrowIfNullOrEmpty(extensionPoolId);
+        ArgumentNullException.ThrowIfNull(body);
 
-        ArgumentNullException.ThrowIfNull(body, nameof(body));
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
 
-        var uri = UriHelper.GetUri($"/api/v2/telephony/providers/edges/extensionpools/{extensionPoolId}", null);
-        var response = await _httpClient.PutAsJsonAsync(uri, body, _options.JsonSerializerOptions, cancellationToken);
+        var response = await client.PutAsJsonAsync($"/api/v2/telephony/providers/edges/extensionpools/{Uri.EscapeDataString(extensionPoolId)}", body, _options.JsonSerializerOptions, cancellationToken);
         response.EnsureSuccessStatusCode();
 
         return await response.Content.ReadFromJsonAsync<ExtensionPool>(_options.JsonSerializerOptions, cancellationToken);
     }
 
     /// <inheritdoc />
 
-    public async Task DeleteExtensionPoolAsync(string extensionPoolId, CancellationToken cancellationToken = default)
+    public async Task<bool> DeleteExtensionPoolAsync(string extensionPoolId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(extensionPoolId);
+
+        var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+
+        var response = await client.DeleteAsync($"/api/v2/telephony/providers/edges/extensionpools/{Uri.EscapeDataString(extensionPoolId)}", cancellationToken);
+        response.EnsureSuccessStatusCode();
+
+        return true;
+    }
+
+    /// <inheritdoc />
+
+    public async Task<EdgeEntityListing> GetEdgesAsync(int? pageSize = null, int? pageNumber = null, string name = null, string siteId = null, string edgeGroupId = null, string sortBy = null, bool? managed = null, CancellationToken cancellationToken = default)
+    {
+        var parameters = new NameValueCollection();
+
+        if (pageSize.HasValue)
+        {
+            parameters.Add("pageSize", pageSize.Value.ToString());
+        }
+
+        if (pageNumber.HasValue)
+        {
+            parameters.Add("pageNumber", pageNumber.Value.ToString());
+        }
+
+        if (!string.IsNullOrEmpty(name))
+        {
+            parameters.Add("name", name);
+        }
+
+        if (!string.IsNullOrEmpty(siteId))
+        {
+            parameters.Add("site.id", siteId);
```

Comment:
Can you check if the parameter is really "site.id"? It seams odd to me. Please compare the old code to ensure the same query parameters are added